Enforce setting the cacert field for secure providers

sgratch opened the pull request:
Fix a bug in which the cacert field was not required if `insecureSkipVerify` is set to false, i.e. provider is secure.
This is reproduced for all providers.

![Screenshot from 2024-03-28 16-55-17](https://github.com/kubev2v/forklift-console-plugin/assets/18169498/4f6dc5e3-b3c2-4a1b-ac6b-d534eeb5e2c1)

yaacov commented:
> cacert field was not required if insecureSkipVerify is set to false

cacert fiels is not required

we have:
a. insecure - insecureSkipVerify set to true
b. secure - insecureSkipVerify set to false, and cacert is empty
b. secure with custom ca - insecureSkipVerify set to false, and cacert is not empty, used as custom ca

sgratch commented:
> > cacert field was not required if insecureSkipVerify is set to false
> 
> cacert fiels is not required
> 
> we have: a. insecure - insecureSkipVerify set to true b. secure - insecureSkipVerify set to false, and cacert is empty b. secure with custom ca - insecureSkipVerify set to false, and cacert is not empty, used as custom ca

1. Is it still supported by backend for all providers? Based on my tests it's not working.
2. It's not documented and we also removed the cacert title that implied that:
  **CA certificate - leave empty to use system CA certificates**and replaced it with only  **CA certificate** title so now this info is only hidden somewhere in a tooltip. The user won't  know it's supported (if still is..).

If we still support this empty cacerrt field for all providers,  I'll update this PR accordingly since it also fix cacert validation for openstack.

sgratch commented:
Need to decide if to handle that or not, based on backend support/fix.
currently it's not clear for which providers it's supported and if working or not.

I'll close this issue for now and let's re-open for 2.7 when this issue will be decided.
